Dockerfile

```diff
@@ -1,14 +1,31 @@
 FROM python:3.8
 LABEL maintainer="GeeksCAT<info@geekscat.org>"
 
+### build-args: ###
+# the name for the non-root user
+ARG USR=anemperfeina
+# the default group of the non-root user
+ARG GRP=anemperfeina
+# the Django environment file. Default the .env.dev.sample
+ARG DOTENV=.env.dev.sample
+
 WORKDIR /anem-per-feina/
 
-COPY requirements.txt ./
+RUN groupadd -r ${GRP} \
+    && useradd --no-log-init -r -g ${GRP} ${USR} \
+    && chown -R ${USR}:${GRP} /anem-per-feina/
+
+COPY --chown=${USR}:${GRP} requirements.txt ./
 
 RUN pip install --no-cache-dir -r requirements.txt
 
-COPY . .
+COPY --chown=${USR}:${GRP} . .
+
+# drop root privileges when running application in container
+USER ${USR}
+
+RUN (cat ${DOTENV}; echo; cat ./docker/secrets.env) > .env
```

Comment:
I would not patch .env definition into the image (as a constant), maybe it's better to define it "live" with `env_file` in our composition.

This image should be used also to deploy our service to k8s, CI, ...